chef/cookbooks/cpe_win_telemetry/recipes/default.rb

```diff
@@ -0,0 +1,16 @@
+#
+# Cookbook Name:: cpe_win_telemetry
+# Recipe:: default
+#
+# vim: syntax=ruby:expandtab:shiftwidth=2:softtabstop=2:tabstop=2
+#
+# Copyright (c) 2017-present, Facebook, Inc.
+# All rights reserved.
+#
+# This source code is licensed under the BSD-style license found in the
+# LICENSE file in the root directory of this source tree. An additional grant
+# of patent rights can be found in the PATENTS file in the same directory.
+#
+
+return unless node.windows? && node.os_at_least?('10.0.15063')
```

Comment:
I think you need to update `cpe_utils` with this PR. The current version of `cpe_utils` in this repo doesn’t support Windows `node.os_at_least`.

Reply:
Good catch, ill update the node.os_at_least function in cpe_utils.  I believe we need both set of checks in the resource and recipe, otherwise we'll get errors, but will double check that.